content/docs/ref/runner.md

```diff
@@ -99,3 +282,61 @@ Any [generic option](/doc/ref) in addition to:
 
    replacing the `IP_ADDRESS` placeholder with the instance address returned by
    `cml runner` (search the output logs for `instanceIp`).
+
+### Using `--cloud-startup-script`
```

Comment:
would swap the `### Using --cloud-ssh-private` and `### Using --cloud-startup-script` section order

Reply:
swapped